### ISSUE TITLE
Все субдомены в пользовательском списке

### ISSUE THREAD

**Issue by blahdy** [gSpotx2f/ruantiblock_openwrt#69]
Ввиду последних новостей есть желание вбить все связанное с *.googlevideo.com в пользовательский список. 

Неуверен, будет ли работать, но вопрос скорее о возможности использовать wildcard. 

Возможно?

**Comment by boozzze** [comment]
Привет, тоже с этой темой влез в issues, имхо, классно было бы, чтобы автор в обнову добавил галочку✔, связанную с этой проблемой замедления , отлаженную и протестированную 🙌😊👌

**Comment by gSpotx2f** [comment]
@blahdy, нет, wildcard в user_entries не работает. IP-адреса добавляются в nftables, а домены в конфиг dnsmasq. Соответственно, для IP-адресов вместо wildcard нужно использовать маски подсетей. Для доменов можно просто добавлять домен более низкого уровня: `googlevideo.com` - означает *.googlevideo.com, т.е. включая субдомены любых уровней (`subdomain.domain.com` работает как *.subdomain.domain.com и т.п. Это правила конфигурации dnsmasq).

@boozzze, новых галочек не нужно, достаточно добавить `googlevideo.com` в user_entries и обновить блэклист.
